mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -114,6 +114,17 @@ MachineGun:
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 
+MachineGunH:
```

Comment:
Probably I'm wrong but I cannot find where `MachineGunH` is actually used (Humvee?).